benchmark-results/build.go

```diff
@@ -0,0 +1,228 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
Should this be under a `cmd/` dir rather than at the top level?

Reply:
Yeah, I want to move it somewhere else. Ideally I'd like to separate it from pgroll itself so that when users import pgroll as a library they don't also need to pull in the charting library. 

To do this I may stick it in somewhere like maybe `/dev/` so indicate that it's for internal use and give it it's own `go.mod` file. WDYT?

Reply:
Could the code to build the benchmarks be a test so that the dependency isn't required outside of test code? That would solve the problem of people having the extra charting dependencies.

Failing that, having it be it's own module could work.

Reply:
I tried moving it all into test code but when I run `go mod tidy` in the root it still pulls in as a direct dependency. I think this would then cause it to be pulled in when users import pgroll. 

I'll have to create a separate module for it.